tests/utils.go

```diff
@@ -2816,6 +2818,34 @@ func WaitForVMICondition(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachi
 	}, time.Duration(timeoutSec)*time.Second, 2).Should(BeTrue(), fmt.Sprintf("Should have %s condition", conditionType))
 }
 
+func matcherVMICondition(conditionType v1.VirtualMachineInstanceConditionType, expected k8sv1.ConditionStatus, options gstruct.Options) types.GomegaMatcher {
+	conditions := func(vmi *v1.VirtualMachineInstance) []v1.VirtualMachineInstanceCondition {
+		return vmi.Status.Conditions
+	}
+	typeMaper := func(obj interface{}) string {
+		condition := obj.(v1.VirtualMachineInstanceCondition)
+		return string(condition.Type)
+	}
+
+	return WithTransform(conditions,
+		gstruct.MatchElements(typeMaper, options,
+			gstruct.Elements{
+				string(conditionType): gstruct.MatchFields(gstruct.IgnoreExtras,
+					gstruct.Fields{
+						"Status": Equal(expected),
+					}),
+			}))
+}
+
+// Must be matched againts slice of conditions
+func MatcherVMIConditionRemovedOrFalse(conditionType v1.VirtualMachineInstanceConditionType) types.GomegaMatcher {
```

Comment:
can we avoid adding code to this bloated file? Can this fit under `tests/framework/matcher/` ?

Reply:
Of course, I was just lazy and wanted to implement it close to the old function.